roles/solrcloud/defaults/main.yml

```diff
@@ -1,48 +1,37 @@
----
-solr_cloud_version: "{{ solr_cloud_download_version | default('7.7.2') }}"
-
-## Service options
-
-# Owner
-solr_user: deploy
-solr_group: deploy
-
-# start on boot
-solr_service_enabled: true
-solr_service_state: started
-
+# defaults file for roles/solrcloud
+solr_cloud_download_version: 8.4.0
+solr_cloud_version: "{{ solr_version }}"
+solr_version: "{{ solr_cloud_download_version }}"
+solr_user: "deploy"
+solr_group: "deploy"
+solr_data_dir: "/solr/data"
+solr_log_dir: /solr/logs
+# ZooKeeper configuration example
+zookeeper_hosts:
+  - "lib-zk1.example.edu:2181"
+  - "lib-zk2.example.edu:2181"
+  - "lib-zk3.example.edu:2181"
+# Solr nodes example
+solr_nodes:
+  - "lib-solr1.example.edu"
+  - "lib-solr2.example.edu"
+  - "lib-solr3.example.edu"
 # Files & Paths
 solr_log4j: "{{ solr_log4j_path | default('/solr/log4j.properties') }}"
-solr_log_dir: /solr/logs
 solr_home: /solr
 solr_data_dir: "{{ solr_home }}/data"
 solr_installation: /opt/solr
-
 # Ports
 solr_port: 8983
 solr_jmx_enabled: "true"
 solr_jmx_port: 1099
 solr_host: "{{ inventory_hostname }}"
-
 # garbage collection settings
-solr_gc_tune: "-XX:NewRatio=3 \
-  -XX:SurvivorRatio=4 \
-  -XX:TargetSurvivorRatio=90 \
-  -XX:MaxTenuringThreshold=8 \
-  -XX:ConcGCThreads=4 \
-  -XX:ParallelGCThreads=4 \
-  -XX:PretenureSizeThreshold=64m \
-  -XX:+ParallelRefProcEnabled"
-
+solr_gc_tune: "-XX:NewRatio=3 -XX:SurvivorRatio=4 -XX:TargetSurvivorRatio=90 -XX:MaxTenuringThreshold=8 -XX:ConcGCThreads=4 -XX:ParallelGCThreads=4 -XX:PretenureSizeThreshold=64m -XX:+ParallelRefProcEnabled"
```

Comment:
I find the old presentation, with line breaks, makes it easier to find what we're setting. We can debate.